suite-native/storage/src/storage.ts

```diff
@@ -16,18 +16,32 @@ export const ENCRYPTED_STORAGE_ID = 'trezorSuite-app-storage';
 export let encryptedStorage: MMKV;
 
 const retrieveStorageEncryptionKey = async () => {
+    let secureKey: string | null = null;
     try {
-        const secureKey = await SecureStore.getItemAsync(ENCRYPTION_KEY);
-
-        if (secureKey) return secureKey;
+        secureKey = await SecureStore.getItemAsync(ENCRYPTION_KEY);
     } catch (error) {
-        // Some users are facing an error when they uninstall the app and then reinstall it,
-        // see https://github.com/expo/expo/issues/23426
-        await SecureStore.deleteItemAsync(ENCRYPTION_KEY);
-        captureException(error);
+        // If there is an error, report it and try to read one more time.
+        captureException(error, { tags: { attempt: 1 } });
+        try {
+            // There were some trouble reading from the SecureStore,
+            // let's wait a bit to make sure it wasn't just temporary error.
+            await new Promise(resolve => setTimeout(resolve, 100));
```

Comment:
I not sure about this, I would try this only if anything else helped, otherwise this will always delay app startup time for 100ms which is quite a lot

Reply:
This is called only if it failed once - it is inside the catch block. For the first try, there is no artificial delay.

Reply:
I did read it wrong, so it's fine